src/ceph-disk/ceph_disk/main.py

```diff
@@ -4524,9 +4531,35 @@ def list_devices():
     return devices
 
 
+def list_zfs():
+    try:
+        out, err, ret = command(
+            [   'zfs',
+                'list',
+                '-o', 'name,mountpoint'
+            ]
+           )
+    except subprocess.CalledProcessError as e:
+        LOG.info('zfs list -o name,mountpoint '
+           'fails.\n (Error: %s)' % e)
+        return 1
```

Comment:
better off raising this exception in this case, `main()` does not use the return value from `func()` or `main_catch()`, and `main_catch()` just ignore the return value of `func()`.